compiler/rustc_middle/src/mir/pretty.rs

```diff
@@ -1896,7 +1896,7 @@ fn pretty_print_const_value_tcx<'tcx>(
         // Aggregates, printed as array/tuple/struct/variant construction syntax.
         //
         // NB: the `has_non_region_param` check ensures that we can use
-        // the `destructure_const` query with an empty `ty::ParamEnv` without
+        // the `destructure_adt_const` query with an empty `ty::ParamEnv` without
```

Comment:
I think this is actually about the `try_destructure_mir_constant_for_user_output` query which internally uses a `TypingEnv::fully_monomorphized`. can you reword this comment to talk about the right query and to say the typingenv instead of "empty ty::ParamEnv`